app/src/main/java/ch/sdp/vibester/util/Util.kt

```diff
@@ -0,0 +1,16 @@
+package ch.sdp.vibester.util
+
+class Util {
+    companion object {
+        fun createNewId(): String {
```

Comment:
consider adding documentation for this (explaining the form of the id, etc)

Reply:
Agreed.